config/initializers/filter_parameter_logging.rb

```diff
@@ -2,5 +2,10 @@
 
 # Be sure to restart your server when you modify this file.
 
-# Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += %i[password api_key token]
+# Configure parameters to be filtered from the log file. Use this to limit dissemination of
+# sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
+# notations and behaviors.
+Rails.application.config.filter_parameters += %i[
+  passw secret token _key crypt salt certificate otp ssn
+  password api_key
+]
```

Comment:
The new application template includes these other default filtered params. Seemed like a good idea to include the lot